index.html

```diff
@@ -1,29 +1,193 @@
+        <h2 class="title">I’m Isaac Pitwa Glad to see you!</h2>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
         <h2 class="label">LET’S CONNECT</h2>
         <ul class="socials">
-            <li class="social-link"><a href="https://twitter.com/isaacpitwa"><img src="./images/icons/twitter.svg" alt="Twitter"/></a></li>
-            <li class="social-link"><a href="https://linkedin.com/in/isaac-pitwa"><img src="./images/icons/linkedin.svg" alt="LinkedIn"/></a></li>
-            <li class="social-link"><a href="https://twitter.com/isaacpitwa"><img src="./images/icons/medium.svg" alt="Medium"/></a></li>
-            <li class="social-link"><a href="https://github.com/isaacpitwa"><img src="./images/icons/github.svg" alt="Github"/></a></li>
-            <li class="social-link"><a href="https://twitter.com/isaacpitwa"><img src="./images/icons/peace.svg" alt="Peace"/></a></li>
+            <li class="social-link"><a href="https://twitter.com/isaacpitwa"><img src="./images/icons/twitter.svg"
+                        alt="Twitter" /></a></li>
+            <li class="social-link"><a href="https://linkedin.com/in/isaac-pitwa"><img src="./images/icons/linkedin.svg"
+                        alt="LinkedIn" /></a></li>
+            <li class="social-link"><a href="https://twitter.com/isaacpitwa"><img src="./images/icons/medium.svg"
+                        alt="Medium" /></a></li>
+            <li class="social-link"><a href="https://github.com/isaacpitwa"><img src="./images/icons/github.svg"
+                        alt="Github" /></a></li>
+            <li class="social-link"><a href="https://twitter.com/isaacpitwa"><img src="./images/icons/peace.svg"
+                        alt="Peace" /></a></li>
 
         </ul>
     </section>
+    <section class="works">
+        <div class="work-card">
+            <img src="/images/project1.svg" alt="Tonic" />
+            <div class="content">
+                <h1 class="project-title">Tonic</h1>
+                <div class="stats">
+                    <h3 class="client">CANOPY</h3>
+                    <div class="dot"></div>
+                    <h3 class="myrole">Back End Dev</h3>
+                    <div class="dot"></div>
+                    <h3 class="year">2015</h3>
+                </div>
+                <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups
+                    required.
+                </p>
+                <div class="languages">
+                    <div class="lang">html</div>
+                    <div class="lang">css</div>
+                    <div class="lang">javaScript</div>
+                </div>
```

Comment:
- [ ] Please let's make these project technologies a `ul` list and each of them a `li` instead of using a `div` tag. The reason is that it's more semantically correct. This equally helps screen readers and other technologies that depend on correct semantics to work.